frontend/src/static/js/v_summary.js

```diff
@@ -1,49 +1,6 @@
-// date functions //
-const DAY_IN_MS = (1000 * 60 * 60 * 24);
-window.DAY_IN_MS = DAY_IN_MS;
-const WEEK_IN_MS = DAY_IN_MS * 7;
+const WEEK_IN_MS = window.DAY_IN_MS * 7;
```

Comment:
Perhaps can put this in the util.js as well? since it's pretty much a constant that might be useful for multiple classes

Reply:
According to previous discussion we decided to move all util function into api.js, I will update the commit message

Reply:
Can place WEEK_IN_MS as a window constant as well

Reply:
noticed after updating the constant is only used in one method. Has shifted it inside the component